webapp/frontend/PHP/classes/Curl.php

```diff
@@ -5,8 +5,8 @@
 class Curl
 {
     public static function searchExamPathForStudents($examID) {
-        $exam_storage_user = "splinter";
-        $exam_storage_password = "1234";
+        $exam_storage_user = getenv('POSTGRES_DB');
```

Comment:
@MateuszKepczynski, please use `EX_STORE_SPLINTER_USER` and `EX_STORE_SPLINTER_PASS` variables for exam_storage authentication
https://github.com/cvlab-ai/splinter/blob/main/.env#L1